Resources/Prototypes/Recipes/Construction/Graphs/utilities/atmos_unary.yml

```diff
@@ -0,0 +1,75 @@
+- type: constructionGraph
+  id: GasUnary
+  start: start
+  graph:
+  - node: start
+    edges:
+    - to: ventpump
+      steps:
+      - material: Steel
+        amount: 2
+        doAfter: 1
+    - to: passivevent
+      steps:
+      - material: Steel
+        amount: 2
+        doAfter: 1
+    - to: ventscrubber
+      steps:
+      - material: Steel
+        amount: 2
+        doAfter: 1
+    - to: outletinjector
+      steps:
+      - material: Steel
+        amount: 2
+        doAfter: 1
+  - node: ventpump
+    entity: GasVentPump
+    edges:
+    - to: start
+      completed:
+      - !type:SpawnPrototype
+        prototype: SheetSteel1
+        amount: 2
+      - !type:DeleteEntity
+      steps:
+      - tool: Welding
+        doAfter: 1
+  - node: passivevent
+    entity: GasPassiveVent
+    edges:
+    - to: start
+      completed:
+      - !type:SpawnPrototype
+        prototype: SheetSteel1
+        amount: 2
+      - !type:DeleteEntity
+      steps:
+      - tool: Welding
+        doAfter: 1
+  - node: ventscrubber
+    entity: GasVentScrubber
+    edges:
+    - to: start
+      completed:
+      - !type:SpawnPrototype
+        prototype: SheetSteel1
+        amount: 2
+      - !type:DeleteEntity
+      steps:
+      - tool: Welding
+        doAfter: 1
+  - node: outletinjector
+    entity: GasOutletInjector
+    edges:
+    - to: start
+      completed:
+      - !type:SpawnPrototype
+        prototype: SheetSteel1
+        amount: 2
+      - !type:DeleteEntity
+      steps:
+      - tool: Welding
+        doAfter: 1
```

Comment:
These might be a little fast for someone to be able to destroy atmos?

Reply:
I don't think so personally as long as it can be built as fast as it can be destroyed. People love tinkering with atmos setups, it's probably best to not make that process too slow?